resources/public/api/conf/2.0/application.yaml

```diff
@@ -0,0 +1,10973 @@
+          "lifeEventId": "6789000002"
+        }
+        </pre>
+                    </td>
+                </tr>
+            </tbody>
+        </table>
+      operationId: requestPropertyPurchaseExtension
+      parameters:
+      - $ref: '#/components/parameters/acceptHeader'
+      - $ref: '#/components/parameters/authorizationHeader'
+      - $ref: '#/components/parameters/lisaManagerReferenceNumber'
+      - $ref: '#/components/parameters/accountId'
+      requestBody:
+        content:
+          application/json:
+            schema:
+              allOf:
+              - $ref: '#/components/schemas/lifetimeIsaManagerAccountsEventsPurchaseExtensionsRequest'
+            examples:
+              example-1:
+                value:
+                  fundReleaseId: '3456789001'
+                  eventDate: 2017-05-10
+                  eventType: Extension one
+              example-2:
+                value:
+                  eventDate: 2017-05-11
+                  eventType: Extension one
+                  supersede:
+                    originalEventDate: 2017-05-10
+                    originalLifeEventId: '6789000001'
+        required: true
+      responses:
+        '201':
+          description: Created
+          content:
+            application/json:
+              schema:
+                allOf:
+                - $ref: '#/components/schemas/lifetimeIsaManagerAccountsEventsPurchaseExtensionsResponse'
+              examples:
+                lifeEventCreated:
+                  value:
+                    status: 201
+                    success: true
+                    data:
+                      message: Extension created
+                      lifeEventId: '6789000001'
+                lifeEventSuperseded:
+                  value:
+                    status: 201
+                    success: true
+                    data:
+                      message: Extension superseded
+                      lifeEventId: '6789000002'
+        '403':
+          description: Forbidden
+          content:
+            application/json:
+              schema:
+                allOf:
+                - $ref: '#/components/schemas/lifetimeIsaErrorResponse'
+              examples:
+                accountClosed:
+                  $ref: '#/components/examples/accountClosed'
+                accountCancelled:
+                  $ref: '#/components/examples/accountCancelled'
+                accountVoid:
+                  $ref: '#/components/examples/accountVoid'
+                extensionNotApproved:
+                  value:
+                    code: FIRST_EXTENSION_NOT_APPROVED
+                    message: A first extension has not yet been approved
+                firstExtensionAlreadyApproved:
+                  value:
+                    code: FIRST_EXTENSION_ALREADY_APPROVED
+                    message: A first extension has already been approved
+                secondExtensionAlreadyApproved:
```

Comment:
This is missing a description. Please add, thanks.

                secondExtensionAlreadyApproved:
                  description: A second extension has already been approved
                  value:
                    code: SECOND_EXTENSION_ALREADY_APPROVED
                    message: A second extension has already been approved